IoT Gateway/Google Cloud Integration/Update JWT and MQTT Agent.py

```diff
@@ -0,0 +1,68 @@
+# -------------------------------------------------------------------------
+# Copyright (c) PTC Inc. and/or all its affiliates. All rights reserved.
+# --------------------------------------------------------------------------
+
+from kepconfig import connection, error
+import kepconfig.iot_gateway as IoT
+from kepconfig.iot_gateway import agent, iot_items
+import datetime
+import jwt
+
+# Agent name and Type to be used - constants from kepconfig.iotgateway 
+# can be used to identify the type of agent
+agent_name = 'IoTCore'
+agent_type = IoT.MQTT_CLIENT_AGENT
+
+# JWT specific variables, Google Cloud Project ID, private key file location and name
+project_id='coastal-throne-266515'
+private_key_file='rsa_private.pem'
+
+# [START iot_mqtt_jwt]
+def create_jwt(project_id, private_key_file, algorithm):
+    
+    token = {
+            # The time that the token was issued at
+            'iat': datetime.datetime.utcnow(),
+            # The time the token expires.
+            'exp': datetime.datetime.utcnow() + datetime.timedelta(minutes=120),
+            # The audience field should always be set to the GCP project id.
+            'aud': project_id
+    }
+
+    # Read the private key file.
+    with open(private_key_file, 'r') as f:
+        private_key = f.read()
+
+    print('Creating JWT using {} from private key file {}'.format(algorithm, private_key_file))
+
+    return jwt.encode(token, private_key, algorithm=algorithm)
+# [END iot_mqtt_jwt]
+
+jwt=create_jwt(project_id, private_key_file, algorithm='RS256').decode("utf-8") # Create JWT and Convert into String
```

Comment:
Recommend moving this below the the function definitions. Helps make the code cleaner to have the functions together prior to the main code running in the script.